att_cybersecurity_alienvault_otx/plugin.spec.yaml

```diff
@@ -67,9 +67,9 @@ types:
       description: File
       required: false
     md5:
-      title: Md5
+      title: MD5
```

Comment:
This will be a major version bump since Md5 is used in the workflow language and workflows that use `{{[Md5]}}` will now need to be updated to use `{{[MD5]}}`